config/perf.rest.server.properties

```diff
@@ -0,0 +1,9 @@
+# rest server
+rest.server.blob.storage.service.factory=com.github.ambry.frontend.AmbryBlobStorageServiceFactory
```

Comment:
[This](https://github.com/linkedin/ambry/wiki/Frontend%20Configuration) is the documentation we have. Anything without a default has to be defined.

I noticed that the link does not have `Router` configs. Will create an issue for that to be added once all router configs are finalized.